docs/sync-storage-handbook/src/README.md

```diff
@@ -0,0 +1,36 @@
+# Introduction
+
+The Application Services libraries provide cross-platform components for storing and syncing user data within the Firefox ecosystem. As we make Firefox available on more platforms, and ship more products that aren't web browsers, we're finding that each product wants to access existing Firefox data, like bookmarks, history, and logins, and make its own data available to others. The goal of the sync and storage components is to expose a uniform, flexible, and high-level way to do this.
+
+* High-level means your application thinks in terms of _what_ it wants to do—"add a bookmark", "store a page visit", "update a saved password", "sync now"—and not _how_. The component takes care of details like defining the database schema, handling migrations, optimizing queries, marshaling and unmarshaling Sync records, and resolving merge conflicts.
+* Uniform means one way to access data everywhere. The same building blocks are used in logins, bookmarks, and history, as well as any custom data types. These are also the same across products and platforms, so you don't need to change three different codebases.
+* Flexible means it's easy for your application to add new data types, evolve its storage schema, and experiment with new ways to represent data.
```

Comment:
This is A+ introduction material.

The "Flexible" part is currently quite aspirational but it's definitely important to our broader story. I'm not sure if "evolve its storage schema" quite fits though, because the previous point talks about how this is taken care of by the component. Perhaps you mean "schema" at a higher level than what the component takes care of?